src/assets/css/Footer.css

```diff
@@ -108,3 +108,72 @@ p {
   width: 20px;
   height: 20px;
 }
+
+.subFooter {
+  border-top: 1px solid #222;
+  font-family: "SFUI", sans-serif;
+  /*display: grid;
+  grid-template-columns: 50% 50%;*/
+  font-size: 10px;
+  color: #7e7e7e;
+  margin: 0px -8px;
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
```

Comment:
what happens with this 3 lines?

Reply:
It aligns components as needed (Here the components are aligned at `baseline`).